hack/.vendor-helpers.sh

```diff
@@ -128,3 +128,12 @@ clean() {
 
 	echo done
 }
+
+fix_rewritten_imports () {
```

Comment:
@dhiltgen a comment explaining what this function is doing would be appreciated :)